### PR TITLE
feat(WEBRTC-119): Documented new ringback property and updated examples.

### PR DESCRIPTION
**Pull request by DeividVeloso**
- Documented new ringback property and updated examples.

## 📝 To Do

- [x] All linters pass
- [x] All tests pass
- [ ] Change documentation based on my changes

## ✋ Manual testing

1. run `npm run build`
2. cd examples/react
3. npm run setup && npm run storybook
4. access examples/react/stories/2-WebDialer.stories.js
5. make a call
6. wait until you listen the ringback sound be played

### Desktop

- [ ] Edge (latest)
- [x] Chrome
- [ ] Firefox
- [ ] Safari

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,15 +41,22 @@ import { TelnyxRTC } from '@telnyx/webrtc';
 
 To initialize the JavaScript SDK, you'll need to authenticate using a Telnyx Connection. You can access the Connection credentials in the [Telnyx Portal](https://portal.telnyx.com/#/app/connections).
 
+To use the `ringbackFile` make sure the option "Generate Ringback Tone" is disabled in your connection configuration on [Telnyx Portal](https://portaldev.telnyx.com/#/app/connections) Inbound tab.
```
Comment (SuaYoo) [inline README.md]:
```suggestion
To use the `ringbackFile`, make sure the "Generate Ringback Tone" option is **disabled** in your [Telnyx Portal connection](https://portaldev.telnyx.com/#/app/connections) configuration (Inbound tab.)
```

File: README.md [hunk 2]
```diff
@@ -41,15 +41,22 @@ import { TelnyxRTC } from '@telnyx/webrtc';
...
   login: username,
   password: password,
-  // Other options
-  // This file can be a wav/mp3 in your local public folder or you can host it in a CDN and pass just the URL, such as https://cdn.company.com/sounds/call.mp3
+  /* 
+    ringtoneFile - This file can be a wav/mp3 in your local public folder or you can host it in a CDN and pass just the URL, such as https://cdn.company.com/sounds/call.mp3
+   */
   ringtoneFile: './sounds/incoming_call.mp3',
+  /*
+    ringbackFile - Used when you disable Generate Ringback Tone to provide your own ringback sound - Enable Instant Ringback (180).
```
Comment (SuaYoo) [inline README.md:57]: Do you need to check the "Enable Instant Ringback (180)" option in the portal in order to use the ringback file?

File: examples/vanilla/index.html [hunk 3]
```diff
@@ -221,6 +221,7 @@ <h5>Remote Video</h5>
           login: document.getElementById('username').value,
           password: document.getElementById('password').value,
           ringtoneFile: './sounds/incoming_call.mp3',
+          // ringbackFile: './sounds/ringback_tone.mp3',
```
Comment (SuaYoo) [inline examples/vanilla/index.html:224]: Did you mean to leave in this comment?

File: README.md (hunk 2 above)
Comment (DeividVeloso) [reply README.md:57]: No, I will remove this.

File: examples/vanilla/index.html (hunk 3 above)
Comment (DeividVeloso) [reply examples/vanilla/index.html:224]: Yes, it is only to show that exists the option.

**Pull request merged**